l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -343,7 +344,21 @@ def setCedentePrestatore(self, company, fatturapa):
     def _setDatiAnagraficiCessionario(self, partner, fatturapa):
         fatturapa.FatturaElettronicaHeader.CessionarioCommittente.\
             DatiAnagrafici = DatiAnagraficiCessionarioType()
-        if not partner.vat and not partner.fiscalcode:
+        if (
```

Comment:
Vedi https://github.com/OCA/l10n-italy/pull/861#discussion_r249776524.
Inoltre si potrebbe controllare che, oltre ad esserci, il `country_id.code` sia diverso da `IT`.